src/StudioCore/Assets/Paramdex/ER/Defs/PlayRegionParam.xml

```diff
@@ -264,9 +264,14 @@
       <Maximum>1</Maximum>
       <SortID>2100</SortID>
     </Field>
-    <Field Def="dummy8 pad1:7">
+    
+    <Field Def="dummy8 pad1_old:7" RemovedVersion="11210015" />
```

Comment:
Is there particular reason to rename old field? Curious, since it's also a relic of original paramdef. Some util I'm guessing is expecting distinct names across versions?